lib/chain.go

```diff
@@ -15,8 +15,8 @@ import (
 
 var lotusPath = "~/.lotus/datastore/chain"
 
-// currently unused but can be used for persisting data
-var entPath = "~/.ent/datastore/chain"
+// persist migrated chain state
+var entChainPath = "~/.ent/datastore/chain"
```

Comment:
Nit: it would be helpful to print both these paths at the time/s that they're used, too.